### PR TITLE
Update monero-core_tr.ts

### PR DESCRIPTION
**Pull request by xmoreee**

### CODE REVIEW

File: translations/monero-core_tr.ts [hunk 1]
```diff
@@ -784,107 +784,107 @@
...
         <location filename="../pages/Receive.qml" line="373"/>
         <source>Create new address</source>
-        <translation type="unfinished"></translation>
+        <translation>Yeni adres oluşturun</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="388"/>
         <source>Set the label of the new address:</source>
-        <translation type="unfinished"></translation>
+        <translation>Yeni adresin etiketini belirleyin</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: Please add `:` at the end of the line

File: translations/monero-core_tr.ts [hunk 2]
```diff
@@ -784,107 +784,107 @@
...
         <location filename="../pages/Receive.qml" line="676"/>
         <source>Enable</source>
-        <translation type="unfinished"></translation>
+        <translation>Etkinleştir</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="697"/>
         <source>Failed to save QrCode to </source>
-        <translation type="unfinished"></translation>
+        <translation>QrKodu kaydetme başarısız oldu.</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: Remove `.` and add whitespace at the end of the line

File: translations/monero-core_tr.ts [hunk 3]
```diff
@@ -1214,155 +1221,155 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="189"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>Yükle</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="205"/>
         <source>Or manually blackball/unblackball a single output:</source>
-        <translation type="unfinished"></translation>
+        <translation>Veya bir çıktıyı manuel olarak oybirliği ile atın veya alın</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: Missing `:`

File: translations/monero-core_tr.ts [hunk 4]
```diff
@@ -1214,155 +1221,155 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="392"/>
         <source>Relative</source>
-        <translation type="unfinished"></translation>
+        <translation>Göreceli</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="408"/>
         <source>Segregation height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ayrım yüksekliği</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: missing `:`

File: translations/monero-core_tr.ts [hunk 5]
```diff
@@ -1761,43 +1768,43 @@ The old wallet cache file will be renamed and can be restored later.
         <location filename="../pages/Transfer.qml" line="567"/>
         <source>
 Number of transactions: </source>
-        <translation type="unfinished"></translation>
+        <translation>İşlem sayısı</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: missing `:` and white space. (the result should be `İşlem sayısı: `)

File: translations/monero-core_tr.ts [hunk 6]
```diff
@@ -1761,43 +1768,43 @@ The old wallet cache file will be renamed and can be restored later.
...
 payment ID: </source>
-        <translation type="unfinished"></translation>
+        <translation>Ödeme No (Payment ID): </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="572"/>
         <source>
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>Miktar</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: missing `:` and white space. Should be `Miktar: ` <- with a space after `:`

File: translations/monero-core_tr.ts [hunk 7]
```diff
@@ -1761,43 +1768,43 @@ The old wallet cache file will be renamed and can be restored later.
...
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="573"/>
         <source>
 Fee: </source>
-        <translation type="unfinished"></translation>
+        <translation>Ücret</translation>
```
Comment (erciccione) [inline translations/monero-core_tr.ts]: missing `:` and white space. Should be `Ücret: ` <- with a space after `:`

**Comment by xmoreee** [comment]
I will do "my best".
In the mean time, here are some things to consider : 

1. Turkish sentence structure differs from English.  For example,
En : Failed to save QrCode to A.
Tr:  QrKodunu A'ya kaydetme başarısız oldu.
So those type of sentence building statements should be coded like
<source>Failed to save QrCode to %1.</source>
So one can correctly translate it as 
<source>QrKodunu %1&apos;ya kaydetme başarısız oldu.</source> ,

2. Some messages (read ones related to key reusing forks) very long and quite confusing for ordinary Joe even in plain English.
I fear some (not tr one :) of their translations would look ridiculous.
I suggest marking those messages as optional in translation.
So that if the translator is not convenient about what s/he can achieve, s/he can skip it,

**Comment by erciccione** [comment]
Thanks for the clarification, but we cannot use `%n` since it would be considered as a place marker.

Some messages can be long and use technical language yes, but our translators did a great job until this moment and most of the translations get reviewed by others. I don't see any reason to make the translation of those strings optional, if translators have problem with the meaning of a string, they usaually ask for clarification on our support channel `#monero-translations`. Please open an issue or suggest changes with a PR if you think something in the original strings should be changed
